hapi-fhir-docs/src/main/resources/ca/uhn/hapi/fhir/changelog/6_1_0/3640-implement-tag-profile-and-security-freetext-search-parameters.yaml

```
@@ -0,0 +1,4 @@
+---
+type: add
+issue: 3640
+title: "Freetext search parameters _tag, _profile and _security were implemented."
```

```suggestion
title: "Extended Lucene/Elasticsearch indexing of search parameters now supports _tag, _profile and _security."
```

We don't call this feature free text.  We call it Extended Lucene/Elasticsearch Indexing.  See https://hapifhir.io/hapi-fhir/docs/server_jpa/elastic.html

Also - can you update that doc page?  Add URI to the list of supported param types, and add _tag, and _security to the token support.

---

Done